src/main/java/com/minecolonies/coremod/client/gui/WindowHireWorker.java

```diff
@@ -202,12 +207,14 @@ private static String createColor(final AbstractBuildingWorker.Skill primary, fi
         return "";
     }
 
+    /* TODO: NEW BLOCKOUT -> make this work
     @Override
     public void onUpdate()
     {
         updateCitizens();
         window.findPaneOfTypeByID(CITIZEN_LIST, ScrollingList.class).refreshElementPanes();
     }
+    */
```

Comment:
What's the issue with this?

Reply:
No longer needed, more like till new Blockout and also breaks provisional button handler